### ISSUE TITLE
cmake error

### ISSUE THREAD

**Issue by chch2010523** [panagelak/Open_Mobile_Manipulator#1]
hi, when I catkin_make follow the "Compiling" section, the following error occurs:

fatal error: exploration_msgs/SetPolygon.h: No such file or directory
 #include <exploration_msgs/SetPolygon.h>

It seems this head file is missing. How to solve this problem?
Thanks a lot!

ubuntu 18.04+ros melodic

**Comment by panagelak** [comment]
Hi,
you can try to build the offending packages seperately or delete them, i
think the frontier exploration smtmes want to be build seperatery and
exploration_server pkg, you may also want to remove the include
moveit_interface from the ommp_moveit_interface header file

Στις Τρί, 28 Ιουλ 2020, 12:31 ο χρήστης chch2010523 <
notifications@github.com> έγραψε:

**Comment by chch2010523** [comment]
Thank you~ I remove these two packages, so the error disappears. Have you ever met this issue before?

**Comment by panagelak** [comment]
Yes i had, the frontier_exploration and exploration_server have some problems when you try to build them in a project i dont know why, you can try to do e.g catkin_make --pkg exploration_server then -> catkin_make. I think i had wrote this issue on the readme. Furthermore frontier exploration in order to work from the official repo, you need to change a little the code bcs it subscribes to smthing like p2/move_base instead of move_base.

That said probably i will fork this packages -> change them to be compatible with my project and mention them to this project as external depedencies

P.S I will fix and the small bug on the ommp_moveit_interface pkg on the header file when i find the time

Sorry i am a little busy

Thanks for opening this issue!

**Comment by chch2010523** [comment]
Thanks a lot!
